spec/support/fixtures/test/main.swift

```diff
@@ -1,3 +1,7 @@
 enum UnusedEnum {
     case unused
 }
+
+class UnusedClass {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/2939388825dc75afe82fef36172d4218aaa27083/spec/support/fixtures/test/main.swift#L5"></span>Class 'UnusedClass' is unused</td>
    </tr>
  </tbody>
</table>